### PR TITLE
Add reason in help menu 

### LINKED ISSUE #423: Update /help to explaing the new funtions of the multiplier comman
after #376 you can give the bot a reason for a multiplier, however when calling the /help command no info on this feature or how to properly syntax the command is given 

image

### PR DESCRIPTION
**Pull request by Draeieg**
Resolves #423 
QA for [423](https://github.com/Draeieg/ubiquibot/issues/2)

### CODE REVIEW

File: bin/index.js
Comment (pavlovcik) [inline bin/index.js:1]: @0xcodercrane I thought we can delete this `bin/index.js` file?

File: bin/index.js [hunk 1]
```diff
@@ -264177,7 +264177,7 @@ exports.userCommands = [
     },
     {
         id: commands_1.IssueCommentCommands.MULTIPLIER,
-        description: `Set bounty multiplier (for treasury)`,
+        description: `Set bounty multiplier (for treasury) can add reason by leaving a space and typing it`,
```
Comment (pavlovcik) [inline bin/index.js:264180]: Maybe it would be best to show an example here?

**Review by pavlovcik — COMMENTED** [review]
Nice pull request!

File: bin/index.js [hunk 2]
```diff
@@ -264177,7 +264177,7 @@ exports.userCommands = [
     },
     {
         id: commands_1.IssueCommentCommands.MULTIPLIER,
-        description: `Set bounty multiplier (for treasury) can add reason by leaving a space and typing it`,
+        description: `Set bounty multiplier (for treasury) can add reason by leaving a space and typing it. Example /multiplier @draeieg 0.5 core contributor on payroll`,
```
Comment (pavlovcik) [inline bin/index.js:264180]:
```suggestion
        description: 'Set bounty multiplier (for treasury) can add reason by leaving a space and typing it. Example: `/multiplier` @draeieg `0.5` "core contributor on payroll"',
```

File: bin/index.js
Comment (0xcodercrane) [reply bin/index.js:1]: Yes, we can

Comment (0xcodercrane) [reply bin/index.js:1]: Let me add `./bin` to `.gitignore` to make sure it's not getting pushed.

Comment (0xcodercrane) [reply bin/index.js:1]: pushed 2 commits.
https://github.com/ubiquity/ubiquibot/commit/84f861da7278fc63af2beaae00de1e9e15b8c979
https://github.com/ubiquity/ubiquibot/commit/48c58dc287514b667aa7e87131fd5419c2a7ce21

**Comment by 0xcodercrane** [comment]
@Draeieg  would you please merge `ubiquity:development` into your `development` branch to resolve conflicts?

**Pull request closed without merging**